src/main/kotlin/uk/gov/justice/digital/hmpps/hmppselectronicmonitoringdataplatformapi/controller/ErrorController.kt

```diff
@@ -0,0 +1,31 @@
+package uk.gov.justice.digital.hmpps.hmppselectronicmonitoringdataplatformapi.controller
+
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.ControllerAdvice
+import org.springframework.web.bind.annotation.ExceptionHandler
+import uk.gov.justice.digital.hmpps.hmppselectronicmonitoringdataplatformapi.model.EmApiError
+
+@ControllerAdvice
+class ErrorController {
+
+  val devMode = true
```

Comment:
Could this be sourced from an environment variable?

Reply:
Yep, that's the plan. But while I didn't know which to use/how that worked, I left it as a const for now.
I expect we'll set it up such that in dev it's true, prod it's false, and preprod probs also false unless we need to do specific preprod debugging.